setup.py

```diff
@@ -45,13 +43,13 @@
         'nbsmoke[all] >=0.4.0',
         'fastparquet >=0.1.6',  # optional dependency
         'holoviews >=1.10.0',
+        'bokeh',
         'pyarrow',
         'netcdf4',
         'twine',   # required for pip packaging
         'rfc3986', # required by twine
         'keyring', # required by twine
-        'spatialpandas',
-        'ipympl'
```

Comment:
Why was this removed? Isn't it needed for the datashader examples now?

Reply:
ipympl is not enabled by default no and I was trying to improve solve time.